### PR TITLE
Addition-of-contact-form-to-portfolio

### PR DESCRIPTION
**Pull request by UsamaMasood12**
In this part of the project, I created a "contact me" form in the footer section of my portfolio. In this way, users can easily contact me.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -196,5 +196,29 @@ <h1>Hey There. <br>
             </div>
         </section>
     </main>
+    <footer id="contact" class="footer">
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure you use the semantic tags on your page for the better structured your HTML is the better SEO your page will be. The `footer` tag doesn't have any semantic meaning here.
`Hint: You may consider using a generic div or better a section tag`
Consider checking [here](https://www.w3schools.com/tags/tag_footer.asp) when to use a footer tag

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @UsamaMasood12 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] In your README, Kindly make sure you have the Get started section where you provided the steps to run the project locally 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -70,7 +78,42 @@
...
+
+### Run tests <a name="run-tests"></a>
+
+npx stylelint "**/*.{css,scss}"
+
+### Deployment <a name="deployment"></a>
+
+You can deploy this project using Chrome Browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (anagudelogu) [inline README.md]: - [ ] Good job implementing this section requested by the previous reviewer 👍🏼 However, if I follow these steps, I won't be able to get a local copy up and running on my machine, these instructions are to run linters.

Please consider updating them! 🤓 🛠️ 

💡 _Hint: You could mention prerequisites as trivial as having a code editor and a browser. For the setup, you could ask to clone your GitHub repo inside a local folder. For installation, you could ask to run `npm install` to install project dependencies. For usage, you could ask to open the `index.html` file on the browser, and so on... At the moment you haven't worked on any tests or deployment, so you can skip those sections for now._

File: index.html [hunk 3]
```diff
@@ -196,5 +196,29 @@ <h1>Hey There. <br>
...
+        <div class="line"><img src="./icons/Rectangle 42.png" alt=""></div>
+    <ul class="logo1">
+        <li><a href="https://github.com/UsamaMasood12"><img src="./icons/C.png" alt=""></a></li>
+        <li><a href="www.linkedin.com/in/usama-masood-b4a35014b"><img src="./icons/I.png" alt=""></a></li>
+        <li><a href="#"><img src="./icons/V.png" alt=""></a></li>
+        <li><a href="https://twitter.com/Usama____Masood"><img src="./icons/T.png" alt=""></a></li>
+        <li><a href="#"><img src="./icons/M.png" alt=""></a></li>
+    </ul>
+    <div class="line2"><img src="./icons/Shape.png" alt=""></div>
+    </section>
```
Comment (anagudelogu) [inline index.html]: Great job implementing the changes required by a previous reviewer! 🚀 

- [ ] **[OPTIONAL]:** I suggest you fix your indentation for the entire file in order to keep it cleaner and improve its readability. 🤓 🛠️

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @UsamaMasood12 👋🏼 
I'm Andres.

You've done a great job implementing some of the requested changes for this milestone 🚀, but there are still some which haven't been addressed yet. You're getting closer, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -90,28 +88,21 @@ To get a local copy up and running, follow these steps.
...
 
-Create a .github/workflows folder and add a copy of .github/workflows/linters.yml to that folder.
+Clone my GitHub repository inside a local folder.
 
 ### Install <a name="install"></a>
 
-Run npm install --save-dev hint@7.x
-npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+Run 'npm install --save-dev hint@7.x' and,
+'npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x' to install project dependencies.
```
Comment (BregornOriginal) [inline README.md]: - [ ] As the previous reviewer mentioned, your readme must have only the instruction to run your project. Linters are added as a dependency on the package.json file. So when a visitor clones your repo and runs `npm install or npm i` linters were automatically installed on his computer. There's no need to give instruction about installing linters, kindly remove it and only add Run `npm install` on the Install section 💯 💯 💯

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @UsamaMasood12  

## Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

one-step-closer-one-more

 ## Highlights

✅ Linters are passing
✅ Good commit history


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @UsamaMasood12, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️
- Linter checks are passing
- Good commit messages
- Required change has been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**